test/integration/test_api.py

```diff
@@ -58,14 +58,14 @@ def mixed_line_endings_file(tmpdir):
 
 def test_py2(py2_file):
 
-    with open(py2_file) as f:
+    with open(py2_file, encoding="utf-8") as f:
```

Comment:
that's the default encoding right? I don't think it's valuable to make it explicit, like we omit 'r' for read mode in the same line.

Reply:
it's a pylint one - it's the default encoding on most systems, but apparently not all, so it's "better" to specify. I've removed tho as your request.

Reply:
this is not obvious if it's better to specify - if some system indeed uses different encoding, it might also have python files in that encoding. Or so it seems.

I have limited trust in pylint suggestions, I think some of it's suggestions are very situational.